mmv1/third_party/terraform/tests/resource_google_project_test.go

```diff
@@ -42,23 +42,25 @@ func testSweepProject(region string) error {
 		return err
 	}
 
+	org := UnsafeGetTestOrgFromEnv()
+	if org == "" {
+		log.Printf("[INFO][SWEEPER_LOG] no organization set, failing project sweeper")
+		return fmt.Errorf("no organization set")
+	}
+
 	token := ""
 	for paginate := true; paginate; {
 		// Filter for projects with test prefix
-		filter := "id:" + testPrefix + "*"
+		filter := fmt.Sprintf("id:\"%s*\" -lifecycleState:DELETE_REQUESTED parent.id:%v", testPrefix, org)
```

Comment:
The quotation marks are what make us check the strict prefixes- `id:tf-test*` would match tf or test showing up anywhere in the string,` id:"tf-test*"` checks for it as a prefix.